app/src/config/constants/sub/paths.js

```diff
@@ -305,10 +305,40 @@ PATHS.SETTINGS.STORAGE_SETTINGS.RELATIVE = PATHS.SETTINGS.RELATIVE + PATHS.SETTI
 PATHS.SETTINGS.STORAGE_SETTINGS.ABSOLUTE = PATHS.SETTINGS.ABSOLUTE + PATHS.SETTINGS.STORAGE_SETTINGS.INDEX;
 
 // Settings - Storage Settings
-PATHS.SETTINGS.DESKTOP_PREFERENCES = {};
-PATHS.SETTINGS.DESKTOP_PREFERENCES.INDEX = "/desktop-preference";
-PATHS.SETTINGS.DESKTOP_PREFERENCES.RELATIVE = PATHS.SETTINGS.RELATIVE + PATHS.SETTINGS.DESKTOP_PREFERENCES.INDEX;
-PATHS.SETTINGS.DESKTOP_PREFERENCES.ABSOLUTE = PATHS.SETTINGS.ABSOLUTE + PATHS.SETTINGS.DESKTOP_PREFERENCES.INDEX;
+PATHS.SETTINGS.DESKTOP_SETTINGS = {};
+PATHS.SETTINGS.DESKTOP_SETTINGS.INDEX = "/desktop-settings";
+PATHS.SETTINGS.DESKTOP_SETTINGS.RELATIVE = PATHS.SETTINGS.RELATIVE + PATHS.SETTINGS.DESKTOP_SETTINGS.INDEX;
+PATHS.SETTINGS.DESKTOP_SETTINGS.ABSOLUTE = PATHS.SETTINGS.ABSOLUTE + PATHS.SETTINGS.DESKTOP_SETTINGS.INDEX;
+
+//Settings - Global Settings
+PATHS.SETTINGS.GLOBAL_SETTINGS = {};
+PATHS.SETTINGS.GLOBAL_SETTINGS.INDEX = "/global-settings";
+PATHS.SETTINGS.GLOBAL_SETTINGS.RELATIVE = PATHS.SETTINGS.RELATIVE + PATHS.SETTINGS.GLOBAL_SETTINGS.INDEX;
+PATHS.SETTINGS.GLOBAL_SETTINGS.ABSOLUTE = PATHS.SETTINGS.ABSOLUTE + PATHS.SETTINGS.GLOBAL_SETTINGS.INDEX;
+
+// Settings - SessionBook Settings
+PATHS.SETTINGS.SESSION_BOOK = {};
+PATHS.SETTINGS.SESSION_BOOK.INDEX = "/sessionbook";
+PATHS.SETTINGS.SESSION_BOOK.RELATIVE = PATHS.SETTINGS.RELATIVE + PATHS.SETTINGS.SESSION_BOOK.INDEX;
+PATHS.SETTINGS.SESSION_BOOK.ABSOLUTE = PATHS.SETTINGS.ABSOLUTE + PATHS.SETTINGS.SESSION_BOOK.INDEX;
```

Comment:
As extension uses `sessions/settings`, we should setup a redirection to `settings/sessionbook`